tests/regression_tests/run_regression_tests.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+#
+# This script will run the umapsort and churn tests to
+# verify that read/write/paging is still working as
+# expected.  These tests currently only test
+# a single file.  As multi-file tests increase (FITS)
+# additional tests will be added.
+#
+churn --directio -f /mnt/intel/regression_test_churn.dat -b 10000 -c 20000 -l 1000 -d 60 
```

Comment:
@mayagokhale, I have added our first regression test script to the umap project.  As soon as I am able to locate a system that we can run our experimental kernel on, we may be able to enable some form of CI.

These two tests are passing now and I'll make sure to run them before submitting subsequent PRs.